### PR TITLE
Add unit tests for participants_file and append_list_as_row

### PR DESCRIPTION
**Pull request by sangfrois**
Adresses #252 by testing new functions in ``bids.py`` and ``utils.py``

## Proposed Changes

  - testing ``append_list_as_row()``
  - testing ``participants_file()``
  
This would lead to increased coverage 👍

### CODE REVIEW

**Comment by sangfrois** [comment]
I'm assigning Eneko to review this PR as we discussed. 

No pressure. I still have to add the tests for the participants file.

File: phys2bids/tests/test_utils.py [hunk 1]
```diff
@@ -95,3 +95,14 @@ def test_load_heuristics():
...
+
+# Test writing rows util
+def append_list_as_row():
+    file_name = 'test_row.tsv'
+    list_of_elem = ["01", 32, 'some_info', 182.98, 'M']
+    utils.append_list_as_row(file_name, list_of_elem)
+    with open(file_name, mode='r') as tsv:
+        tsv_read = reader(tsv, delimiter="\t")
+        for row in tsv_read:
+            assert len(row) == len(list_of_elem)
```
[the remark this answers is not in the document]
Comment (sangfrois) [reply phys2bids/tests/test_utils.py]: this should be addressed as of now

**Comment by sangfrois** [comment]
Both new functions should be checked and up to date with master. I'm not sure about the last tests in ``test_bids.py`` - Waiting for feedback.

**Comment by eurunuela** [comment]
@sangfrois I added the remaining test. It was just a matter of repeating the last one you wrote to check that the subject was already in the file.

I don't know why Travis is complaining about numpy today. It probably has something to do with the lowest numpy version we support. Anyway, the test pass and coverage increases significantly, so good job! 🙌🏻

**Pull request merged**